client/src/main/scala/ch/epfl/bluebrain/nexus/iam/client/types/events/decoders.scala

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.bluebrain.nexus.iam.client.types.events
+
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.iam.client.types.AccessControlList.aclEntityDecoder
+import ch.epfl.bluebrain.nexus.iam.client.types.GrantType.Camel._
+import ch.epfl.bluebrain.nexus.iam.client.types.Identity.Subject
+import ch.epfl.bluebrain.nexus.iam.client.types.{AccessControlList, Identity, Permission}
+import ch.epfl.bluebrain.nexus.rdf.instances._
+import ch.epfl.bluebrain.nexus.rdf.Iri
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto._
+import io.circe.{Decoder, DecodingFailure}
+
+object decoders {
```

Comment:
I think we can safely put the decoders in the companion object of the events since the format is fixed.